### PR TITLE
Support multi-label setup in Text Classification Taskflow

### PR DESCRIPTION
**Pull request by sijunhe**
### PR types
Function optimization

### PR changes
Others

### Description
- support multi-label setup in Text Classification Taskflow

### CODE REVIEW

File: paddlenlp/taskflow/text_classification.py [hunk 1]
```diff
@@ -135,40 +170,58 @@ def _run_model(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
...
+                    scores = F.softmax(logits, axis=-1).numpy()
+                    labels = paddle.argmax(logits, axis=-1).numpy()
+                else:  # static graph
+                    scores = np_softmax(logits, axis=-1)
+                    labels = np.argmax(logits, axis=-1)
+                for score, label in zip(scores, labels):
+                    postprocessed_output = {}
+                    postprocessed_output["predictions"] = [{"label": self.id2label[label], "score": score[label]}]
+                    postprocessed_outputs.append(postprocessed_output)
+            else:  # multi_label
```
Comment (wawltor) [inline paddlenlp/taskflow/text_classification.py:210]: 这里是不是elif model == "multi_label" 更好一点？ 后面需要考虑一下层次分类接入

**Review by wawltor — APPROVED** [review]
LGTM
